test/SlowTests/Server/Documents/ETL/Snowflake/SnowflakeEtlTests.cs

```diff
@@ -0,0 +1,1374 @@
+            await exportOperation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+            var operation = await dstStore.Smuggler.ImportAsync(new DatabaseSmugglerImportOptions(), exportFile);
+            await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+
+            var destinationRecord =
+                await dstStore.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(dstStore.Database));
+            Assert.Equal(1, destinationRecord.SnowflakeEtls.Count);
+            Assert.Equal(1, destinationRecord.SnowflakeConnectionStrings.Count);
+
+            Assert.Equal(DefaultScript, destinationRecord.SnowflakeEtls[0].Transforms[0].Script);
+            Assert.Equal(["Orders"], destinationRecord.SnowflakeEtls[0].Transforms[0].Collections);
+            Assert.Equal("OrdersAndLines", destinationRecord.SnowflakeEtls[0].Transforms[0].Name);
+
+            Assert.Equal(2, destinationRecord.SnowflakeEtls[0].SnowflakeTables.Count);
+            Assert.Equal("Orders", destinationRecord.SnowflakeEtls[0].SnowflakeTables[0].TableName);
+            Assert.Equal("OrderLines", destinationRecord.SnowflakeEtls[0].SnowflakeTables[1].TableName);
+            
+            Assert.Equal("Id", destinationRecord.SnowflakeEtls[0].SnowflakeTables[0].DocumentIdColumn);
+            Assert.Equal("OrderId", destinationRecord.SnowflakeEtls[0].SnowflakeTables[1].DocumentIdColumn);
+            
+            Assert.False(destinationRecord.SnowflakeEtls[0].SnowflakeTables[0].InsertOnlyMode);
+            Assert.False(destinationRecord.SnowflakeEtls[0].SnowflakeTables[1].InsertOnlyMode);
+        }
+    }
+
+    [RequiresSnowflakeFact]
+    public async Task SnowflakeConnectionStringAuditJsonShouldnNotContainCredentials()
+    {
+        SnowflakeConnectionString cs = new() { ConnectionString = "secret", Name = "Test" };
+        var auditJson = cs.ToAuditJson();
+        Assert.False(auditJson.Properties.Select(x => x.Name).Contains("ConnectionString"));
+    }
+
+    internal static long GetOrdersCount(string connectionString)
+    {
+        using (var con = new SnowflakeDbConnection())
+        {
+            con.ConnectionString = connectionString;
+            con.Open();
+
+            using (var dbCommand = con.CreateCommand())
+            {
+                dbCommand.CommandText = " SELECT COUNT(*) FROM Orders";
+                return (long)dbCommand.ExecuteScalar();
+            }
+        }
+    }
+    
+    private class Order
+    {
+        public Address Address { get; set; }
+        public string Id { get; set; }
+        public List<OrderLine> OrderLines { get; set; }
+    }
+
+    private class Address
+    {
+        public string City { get; set; }
+    }
+
+    private class OrderLine
+    {
+        public string Product { get; set; }
+        public int Quantity { get; set; }
+        public int Cost { get; set; }
+    }
+
+    private class FavouriteOrder : Order
+    {
+
+    }
+
+}
```

Comment:
Did I miss something or there is no test for export / import?

Reply:
there's `ShouldImportTask()`